### ISSUE TITLE
APA WG comment: semantic layers

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/imsc#524]
[Section C. Forced content](https://www.w3.org/TR/2019/WD-ttml-imsc1.2-20191128/#forced-content) This seems like a temporary solution to us. Mightn't it be better to define semantic layers of information so that each layer could be made visible and invisible at runtime as appropriate for the user?  For example, the user might want to see either speech-only (subtitles), (parts of subtitles) or some combination of these.

**Comment by gzimmermann** [comment]
I think engaging into a conversation with TTWG makes sense. We need to develop meaningful use cases and clearly state the purpose of a semantic layering.

**Comment by gzimmermann** [comment]
I agree. This issue needs more time than we have currently available for IMSC 1.2.
We should arrange for a joint meeting after IMSC 1.2 was published.

**Comment by michael-n-cooper** [comment]
APA agrees to defer this to future version.

**Comment by michael-n-cooper** [comment]
Re-adding a11y-needs-resolution label because of new information about how the horizontal review tracker works, but this is still signed off from APA.

**Comment by nigelmegitt** [comment]
Discussed in joint call with APA, TTWG and MEIG today 2024-09-27. Further discussion needed; in the meantime, a note pointing to the potential use of DAPT metadata for `represents` and language source would be helpful.

**Comment by palemieux** [comment]
I am not excited to introduce additional metadata without a matching track selection algorithm, such as the one specified in [Common Manifest](https://www.movielabs.com/md/practices/manifest/ManifestPractices_TrackSelection_v1.0.pdf).

**Comment by nigelmegitt** [comment]
Same, as a normative requirement. The idea of the note is to cross-pollinate knowledge of metadata that's available that would be permitted in IMSC, but not required. It formally already is permitted I believe, so this would be informative only.
